services/API-service/src/api/event/event.controller.ts

```diff
@@ -165,24 +165,17 @@ export class EventController {
 
   @UseGuards(RolesGuard)
   @Roles(UserRole.DisasterManager)
-  @ApiOperation({ summary: 'Stop trigger for given admin-area.' })
+  @ApiOperation({ summary: 'Set trigger for event areas.' })
```

Comment:
event areas is a bit of a complicated term, since we have also reserved that for the one-per-event-polygons as defined in event-area.service (see also glossary). I know that we use 'event area' already as you mean it here, also in other places in event.service but we should clear this up from now on if possible.

Reply:
keeping this open, so I can check the change later

Reply:
@jannisvisser should I change the string back to admin area? or how do we clear this up?

Reply:
I'm also not sure, but maybe 'event admin-area' is at least better.

Reply:
I committed this